identity/src/main/java/com/android/identity/PresentationHelper.java

```diff
@@ -320,7 +323,12 @@ public void startListening(
                         new DataTransportBlePeripheralServerMode(mContext, mLoggingFlags);
                 bleTransport.setServiceUuid(serviceUuid);
                 bleTransport.setUseL2CAPIfAvailable(useL2CAPIfAvailable);
+                bleTransport.setClearCache(bleClearCache);
```

Comment:
Why are we setting this hear? The log line below says it's ignored.

Reply:
It seems more odd to me to have the log message _and_ the code setting it.

If you're concerned with people being confused, maybe add a comment?

Reply:
Removed in latest PS.